http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -367,6 +367,22 @@ public void testRequestObjectConfigMissingENCValue() throws Exception {
         testRequestObjectInvalidConfiguration(getOidcConfigurationInputStreamWithoutEncValue(REQUEST.getValue(), RSA_OAEP), RequestObjectErrorType.MISSING_ENC_VALUE);
     }
 
+    @Test
+    // Generate an invalid sessionRandomValue so that the nonce check fails
+    public void testSessionIdNonceMismatch() throws Exception {
+        performAuthentication(
+                getOidcConfigurationInputStreamWithRequestParameter(REQUEST.getValue(), NONE, "", ""),
+                KeycloakConfiguration.ALICE, KeycloakConfiguration.ALICE_PASSWORD,true,
+                HttpStatus.SC_FORBIDDEN,null, CLIENT_PAGE_TEXT, true);
+    }
+
+    @Test
+    public void testtestOpenIDScopeNONCE() throws Exception {
```

Comment:
s/testtest/test

We should still be testing the mismatch case here.

Reply:
done